bloc-did-method.md

```diff
@@ -8,99 +8,165 @@ The `did:bloc` DID method allows groups of independent organizations to share cu
+- The stakeholder config cache is removed
 
-##### Stakeholder Configurations
-Within `stakeholders/` are files for stakeholder configs.
+Verification: The signatures array must be removed from the config file and the config file must be canonicalized (as done by JWS<!--does jws even canonicalize or does it sign a particular string representation...-->) before the signatures are verified against the file.
 
-There must be at least one stakeholder config file here, for the consortium domain to be usable for resolving DIDs.
-
-Each of these files is named `[hash].json`, where `hash` is the SHA-256 hash of the contents of the file. This hash value is the value used in the consortium config to link to this stakeholder config.
+##### Stakeholder Config Files
+Each of these files is named `[domain].json`, where `[domain]` is the URL domain, owned by the stakeholder organization, where you can find the canonical copy of the stakeholder's configuration.
 
 A stakeholder configuration file contains:
 - The stakeholder's domain
+- The stakeholder's DID (did:bloc)
+  - TODO: also include a did:web DID doc? Same DID, same verkeys, used for bootstrapping.
 - Stakeholder custom configuration settings
 - The stakeholder's Sidetree endpoints
-- A detached JWS signature on the file, created by the stakeholder
-- a local hashlink to the previous version of this config file
+- the hash of the previous version of this config file
 
-TODO: Specify how the stakeholders are identified as the owners of the signing keys. For example, they could have DIDs, with the docs resolvable in the consortium (with some appropriate method of bootstrapping trust), or they could have x.509 certs.
+TODO: this should probably be in a secondary file, or stored in a way that's easy to detach without risk of screwing up the payload text
+- A detached JWS signature on the file, created by the stakeholder
 
 ##### History
 The `history/` directory contains historical consortium configs. Each such file is named `[hash].json`, where `[hash]` is the SHA-256 hash of the given file.
 
+TODO: instead of separate history files, have a complete history database file, as either an object or a concat JSON?
+
+TODO: consortium history may need to cache the stakeholder configs and verkeys - since these can be replaced on the stakeholder domain (or the stakeholder could disappear completely), with the associated consortium configs being subsequently unverifiable.
+
+
+##### Stakeholder Files
+    .well-known/did-bloc/
+        [domain].json
+        history/
+
+`[domain].json` is the stakeholder configuration file for this stakeholder.
+
+`history/` is a directory of previous stakeholder configs for this stakeholder
+
+TODO: is a history for the stakeholder config necessary? If the consortium caches historical stakeholder info - specifically, DID -> verkey mappings so historical consortium files can be verified, then the stakeholders don't need to keep historical data, or even exist any longer.
+
 <!--
```

Comment:
Suggest either leaving the TODO visible in the rendered text or creating issues for them